tests/cypress/e2e/for-teacher_page/class_page/other_buttons.cy.js

```diff
@@ -0,0 +1,56 @@
+import {loginForTeacher} from '../../tools/login/login.js'
+import { navigateToClass } from "../../tools/classes/class.js";
+import { goToTeachersPage } from '../../tools/navigation/nav.js';
+
+beforeEach(() => {
+  // do we want this test to be run for a second teacher as well?
```

Comment:
Do we want the whole other_buttons test, to also be tested for a second teacher?

Reply:
I think so. It'll make the test slower (and I think you'd need to make a fresh class before each set of test: one for the teacher and one for the second teacher), but tests are so useful already that I think the added time is worth it! The same answer goes for every test where this question was raised.